packages/core/content-releases/admin/src/pages/ReleaseDetailsPage.tsx

```diff
@@ -635,20 +635,20 @@ const ReleaseDetailsBody = () => {
                 <Table.Body>
                   {releaseActions[key].map(({ id, type, entry }) => (
                     <Tr key={id}>
-                      <Td width={'25%'}>
+                      <Td width="25%" maxWidth="200px">
```

Comment:
I think instead of the maxWidth you could try applying `table-layout: fixed` to the table element

Reply:
Actually I'm not sure it's a good idea it makes the table even more unresponsive then it already is

Reply:
Doing this I lose the alignment and I have other ui issues